src/commands/load.js

```diff
@@ -1,9 +1,33 @@
-import Player from '../player/Player.js';
-import DragonsBite from '../weapons/DragonsBite.js';
+import * as dotenv from 'dotenv';
+dotenv.config();
 
-export const load = (ctx) => {
-  ctx.player = new Player;
-  const weapon = new DragonsBite;
-  ctx.player.skills = weapon.skills;
-  return `Your active adventurer is ${ctx.player.name}.`;
-};
+import { MessageEmbed } from 'discord.js';
+import Player from '../models/Player.js';
+import connection from '../database/connection.js';
+
+const { COMMAND_PREFIX } = process.env;
+
+export const load = (ctx, name) => connection(async (db) => {
+  const { id, username } = ctx.msg.author;
+  const player = await Player.load(id, db);
+  if (!player) {
+    return `You have not signed up. Please create an account with the \`${COMMAND_PREFIX}signup\` command.`;
+  }
```

Comment:
This gets repeated all over the place, kind of a pain, and will lead to problems when we need to change it and have to hunt it down in every command. #19 will get rid of it and make everything better!

We also have to do the dotenv in any file where we access `process.env`, otherwise we have no guarantee that the dotenv is loaded before attempting to access the env. I'll fix this another time.